runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -231,8 +231,10 @@ ClassFileOracle::ClassFileOracle(BufferManager *bufferManager, J9CfrClassFile *c
 	_hasNonStaticFields(false),
 	_hasNonEmptyConstructor(false),
 	_preloadAttribute(NULL),
-	_implicitCreation(NULL),
 #endif /* J9VM_OPT_VALHALLA_VALUE_TYPES */
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+	_implicitCreation(NULL),
+#endif /* J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES */
```

Comment:
This comment should repeat the full `#if` condition:
```c
#endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
```
Fixing line 234 also would be welcome.